libbeat/cmd/instance/beat.go

```diff
@@ -490,10 +490,10 @@ func (b *Beat) Setup(settings Settings, bt beat.Creator, setup SetupSettings) er
 				m := b.IdxSupporter.Manager(idxmgmt.NewESClientHandler(esClient), idxmgmt.BeatsAssets(b.Fields))
 				var tmplLoadMode, ilmLoadMode = idxmgmt.LoadModeUnset, idxmgmt.LoadModeUnset
 				if setup.Template {
-					tmplLoadMode = idxmgmt.LoadModeForce
+					tmplLoadMode = idxmgmt.LoadModeOverwrite
 				}
 				if setup.ILMPolicy {
-					ilmLoadMode = idxmgmt.LoadModeForce
+					ilmLoadMode = idxmgmt.LoadModeOverwrite
```

Comment:
`LoadModeOverwrite` is used by `setup`, while `LoadModeForce` is used by `export`. As described in the LoadMode definitions: 
> LoadModeOverwrite indicates overwriting existing components, if loading is not generally disabled.
> LoadModeForce indicates forcing to load components in any case, independent of general loading configurations.

`export` always prints the template and ilm policies, not matter what is configured, while `setup` only operates if `setup.template.enabled` or `setup.ilm.enabled` is set accordingly. 
This is aligned with current behavior in `7.0.0`. No semantic change here. The tests are aiming to point out this behavior, as it can be quite confusing.